lightning/src/ln/channelmanager.rs

```diff
@@ -1853,13 +1864,35 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 		events.append(&mut new_events);
 	}
 
+	/// Free the background events, generally called from timer_chan_freshness_every_min.
+	///
+	/// Exposed for testing to allow us to process events quickly without generating accidental
+	/// BroadcastChannelUpdate events in timer_chan_freshness_every_min.
```

Comment:
Rather than making this `pub(crate)` define a testing-only method that calls this if you don't want to expose it to the rest of the crate.